components/x-gift-article/src/HighlightCheckbox.jsx

```diff
@@ -2,19 +2,21 @@ import { h } from '@financial-times/x-engine'
 
 export default ({ includeHighlightsHandler, includeHighlights, isGiftUrlCreated }) => {
 	return (
-		<label htmlFor="includeHighlights">
-			<input
-				type="checkbox"
-				id="includeHighlights"
-				name="includeHighlights"
-				value={includeHighlights}
-				checked={includeHighlights}
-				onChange={includeHighlightsHandler}
-				disabled={isGiftUrlCreated}
-			/>
-			<span className="o-forms-input__label x-gift-article__checkbox-span">
-				Make highlights visible to recipients
-			</span>
-		</label>
+		<div className="o-forms-input o-forms-input--checkbox o-forms-field x-gift-article__checkbox">
```

Comment:
I just moved code about the checkbox into a separate file. Probably @MMSaleh-FT can provide more info.

Reply:
@jkerr321  i remember that before i write this code i went through the x-gift-article code to see how it was done in other components and this is the form construction pattern that i found used in

Reply:
@MMSaleh-FT this is the comment Anna and I were chatting about in stand up

Reply:
Ahh sorry Mo didn't see you'd already replied! Ok if that's how it's been done elsewhere then fine 👍

Reply:
I think that we can just merge this now since the UI is good then we can update the form construction in a new PR after the release since there should be another PR for tracking we can update this in it

Reply:
No I think it's fine - if it's done like this elsewhere already then we don't need to change it for now, let's merge 👍